daq/cpn_state_collector.py

```diff
@@ -0,0 +1,38 @@
+"""Collecting the state of CPN components"""
+
+import copy
+import logging
+import os
+import os.path
+import yaml
+
+LOGGER = logging.getLogger('forch')
+
+KEY_NODES = 'cpn_nodes'
+KEY_ATTRIBUTES = 'attributes'
+
+
+class CPNStateCollector:
+    """Processing and storing CPN components states"""
+    def __init__(self):
+        self._cpn_state = {}
+        self._nodes_state = self._cpn_state.setdefault(KEY_NODES, {})
+
+        cpn_file_name = os.getenv('CPN_YAML_FILE', '')
+        if os.path.isfile(cpn_file_name):
```

Comment:
Add some logging about what's going on here. E.g. if no file is specified, it should be obvious. If it is specified, then it should say what it's using.

Reply:
Done